packages/cli/src/commands/ci/config/unset.ts

```diff
@@ -0,0 +1,36 @@
+import {Command, flags} from '@heroku-cli/command'
+import {Args, ux} from '@oclif/core'
+import {getPipeline} from '../../../lib/ci/pipelines'
+import {setConfigVars} from '../../../lib/ci/heroku-api'
+
+export default class CiConfigUnset extends Command {
+  static description = 'unset CI config vars'
+  static topic = 'ci'
+  static examples = [
+    '$ heroku ci:config:unset RAILS_ENV',
+  ]
+
```

Comment:
Original command has `variableArgs: true,` and needs:
```suggestion

static strict = false

```